client/scripts/views/modals/manage_community_modal/community_metadata_management_table.ts

```diff
@@ -48,131 +56,178 @@ m.Component<IChainOrCommMetadataManagementAttrs, ICommunityMetadataManagementSta
-      }),
-      m(InputPropertyRow, {
-        title: 'Domain',
-        defaultValue: vnode.state.customDomain,
-        placeholder: 'Contact support', // gov.edgewa.re
-        onChangeHandler: (v) => { vnode.state.customDomain = v; },
-        disabled: true, // Custom domains should be admin configurable only
-      }),
-      m(InputPropertyRow, {
-        title: 'Terms of Service',
-        defaultValue: vnode.state.terms,
-        placeholder: 'Url that new users see',
-        onChangeHandler: (v) => { vnode.state.terms = v; },
-      }),
-      m(TogglePropertyRow, {
-        title: 'Privacy',
-        defaultValue: vnode.attrs.community.privacyEnabled,
-        onToggle: (checked) => { vnode.state.privacyEnabled = checked; },
-        caption: (checked) => checked ? 'Threads are private to members' : 'Threads are visible to the public',
-      }),
-      m(TogglePropertyRow, {
-        title: 'Invites',
-        defaultValue: vnode.attrs.community.invitesEnabled,
-        onToggle: (checked) => { vnode.state.invitesEnabled = checked; },
-        caption: (checked) => checked ? 'Anyone can invite new members' : 'Admins/mods can invite new members',
+    return m('.CommunityMetadataManagementTable', [
+      m(AvatarUpload, {
+        avatarScope: AvatarScope.Community,
+        uploadStartedCallback: () => {
+          vnode.state.uploadInProgress = true;
+        },
+        uploadCompleteCallback: (files) => {
+          files.forEach((f) => {
+            if (!f.uploadURL) return;
+            const url = f.uploadURL.replace(/\?.*/, '');
+            vnode.state.iconUrl = url;
+            $((vnode as any).dom)
+              .find('input[name=avatarUrl]')
+              .val(url.trim());
+          });
+          vnode.state.uploadInProgress = false;
+        },
       }),
-      m('tr', [
-        m('td', 'Admins'),
-        m('td', [ m(ManageRolesRow, {
-          roledata: vnode.attrs.admins,
-          onRoleUpdate: (oldRole, newRole) => { vnode.attrs.onRoleUpdate(oldRole, newRole); },
-        }), ]),
-      ]),
-      vnode.attrs.mods.length > 0
-        && m('tr', [
-          m('td', 'Moderators'),
-          m('td', [ m(ManageRolesRow, {
-            roledata: vnode.attrs.mods,
-            onRoleUpdate: (oldRole, newRole) => { vnode.attrs.onRoleUpdate(oldRole, newRole); },
-          }), ])
-        ]),
-    ]),
-    m(Button, {
-      label: 'Save changes',
-      intent: 'primary',
-      rounded: true,
-      onclick: async (e) => {
-        const {
-          name,
-          description,
-          iconUrl,
-          website,
-          discord,
-          element,
-          telegram,
-          github,
-          stagesEnabled,
-          customStages,
-          customDomain,
-          terms,
-          invitesEnabled,
-          privacyEnabled,
-        } = vnode.state;
-        try {
-          await vnode.attrs.community.updateCommunityData({
+      m(
+        Table,
```

Comment:
Can we fix things like this? And also update the prettier file so this doesn't happen? @gdjohnson @jnaviask

Reply:
Yeah we can just disable the rule in .eslintrc or .prettierrc.